packages/shopping-cart/src/types.ts

```diff
@@ -645,6 +645,7 @@ export interface ResponseCartProductExtra {
 	is_marketplace_product?: boolean;
 	product_slug?: string;
 	product_type?: 'marketplace_plugin' | 'marketplace_theme' | 'saas_plugin';
+	hideProductVariants?: boolean;
```

Comment:
Its name is pretty self explanatory, but would you mind adding a short comment block above this property? I'm trying to eventually get a description for everything here in preparation for eventually validating them all.

Reply:
Thanks for chiming in @sirbrillig! Happy to take care of this now.

Reply:
Added some comments in [cb92b12](https://github.com/Automattic/wp-calypso/pull/94680/commits/cb92b129fba49f19166f8634fb7624c6a52c9173). I'm merging this PR early because of a hard deadline on the 20th and requests from marketing to test this live asap.

With that being said, I'm happy to make any updates if you have any feedback on the inline comment 🙂